projects/plugins/jetpack/_inc/client/my-plan/my-plan-header/index.js

```diff
@@ -11,13 +11,15 @@ import { find, isEmpty } from 'lodash';
  */
 import { createInterpolateElement } from '@wordpress/element';
 import { __, _n, _x, sprintf } from '@wordpress/i18n';
+import { getRedirectUrl } from '@automattic/jetpack-components';
```

Comment:
Not going to block based on this, but I'd think should be an external dep, not a WP one.